src/index.js

```diff
@@ -16,14 +16,22 @@ import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 
 import './utils/polyfills';
-import store from './store';
+import { configureStore } from './store';
+import { getRunsWebSocketURL } from './api';
 
 import App from './containers/App';
 
+const runsWebSocket = new WebSocket(getRunsWebSocketURL());
+function closeSocket() {
+  runsWebSocket.close();
+}
+
+const store = configureStore({ runsWebSocket });
```

Comment:
should probably just call this `webSocket` since it'll handle more than runs

Reply:
mind if i tackle that in the next ? Going to change the api too

Reply:
works for me 👍